examples/tests/test_examples.py

```diff
@@ -13,7 +13,7 @@
 import pytest
 
 
-from testutils import (
+from tests.testutils import (
```

Comment:
With this change I can no longer run `pytest examples` from the project root. But CI apparently still can. 🤔  

The bare `import testutils` should work because we modify `sys.path` in `root/conftest.py`. Did this not work for you?

Reply:
It works with or without for me. But when I have it with my language server gets it, that's why I changed it. 
Since we basically starting from the same testing scripts for shadertoy, I looked at what changes we made there - and this was one of them. Can be reverted if you prefer.

Reply:
Nevermind. It failed for me because my env is being weird.

Reply:
... on second thought. The problem is that I have pygfx installed with `pip install -e .`, which causes the pygfx root dir to be added to `sys.path`, so when `import tests`, it takes the `pygfx/tests` instead of `./tests`. I'll come back to this 😄

Reply:
> I'll come back to this

In another pr.